buildozer/runner.bash.template

```diff
@@ -2,11 +2,26 @@
 
 set -euo pipefail
 
-BUILDOZER_SHORT_PATH=@@BUILDOZER_SHORT_PATH@@
```

Comment:
This line movement is due to sorting this block of variables. I can undo this if you'd prefer to keep it in a separate commit.